src/stories/2-frontend-engine/conditional-rendering/conditional-rendering-number.stories.tsx

```diff
@@ -169,3 +169,63 @@ Integer.args = {
 		validation: [{ required: true }],
 	},
 };
+
+export const Positive = Template.bind({});
```

Comment:
I think this is introduced from a merge conflict, can remove the duplicated stories